src/Microsoft.Framework.DependencyInjection/ServiceLookup/ServiceScope.cs

```diff
@@ -19,9 +21,19 @@ public IServiceProvider ServiceProvider
             get { return _scopedProvider; }
         }
 
+        internal void Reset()
```

Comment:
Not sure what to write... Its a method that is explicitly bonded to the pooling of ServiceScopeFactory so if the class were made public rather than internal for whatever reason the exposed api would be for it to function as a standalone class (which is why I left public constructor in); whereas the "internals" only work as a class pair.

Not hugely fussed on visibility though, can make public :)